### PR TITLE
add the home page

### PR DESCRIPTION
**Pull request by Roland-Sankara**
### What does this PR solve?
This pr adds an HTML and CSS file for the homepage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Peer Learning</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+
```
Comment (Yesigomwe) [inline index.html:7]: Please align your code as expected.

Comment (Roland-Sankara) [reply index.html:7]: Boss, I think you don't see well the code is aligned

**Comment by Yesigomwe** [comment]
Looks good to me

**Comment by Yesigomwe** [comment]
looks good to me

**Review by Yesigomwe — APPROVED** [review]
looks good to me

**Pull request merged**